src/data-dictionary/events/ContainerSample/memoryUsageBytes.md

```diff
@@ -6,4 +6,6 @@ events:
   - ContainerSample
 ---
 
-The total memory the container is using.
\ No newline at end of file
+The amount of memory the container is using. This memory doesn't include swap.
+
+**Breaking change**: This metric includes memory + swap on versions v1.5.0 or higher of `nri-docker`.
```

Comment:
```suggestion
**Breaking change**: This metric used to include swap memory `nri-docker` v1.5.0 and earlier (Agent `1.16.x` and earlier).
```

⬆️ Users probably do not know which version of `nri-docker` they have since it is bundled with the agent.